src/server/check-answers/index.njk

```diff
@@ -21,35 +21,11 @@
       </div>
     </div>
   </div>
-{% endblock %}
-{% block questions %}
   <h2 class="govuk-heading-m">Your declaration</h2>
-  <!-- form URL below turns off errors on the next page  -->
-  {{
-    govukCheckboxes
-    ({
-      name: "confirmation",
-      id: "confirmation",
-      errorMessage: errors.confirmation,
-      fieldset: {
-        legend:{
-          text: "Before you submit your application, you must select one of the declaration check boxes.",
-          isPageHeading: false
-        }
-      },
-      items: [{
-        value: "confirm",
-        text: "I confirm all the information given about my farm or premises is correct to the best of my knowledge",
-        attributes: {
-          'data-testid': 'confirm-statements-checkbox'
-        }
-      }, {
-        value: "other",
-        text: "I am submitting this form on behalf of someone else and confirm all the information given is correct to the best of my knowledge",
-        attributes: {
-          'data-testid': 'someone-else-checkbox'
-        }
-      }]
-    })
-  }}
 {% endblock %}
+{%
+  set viewModelOptions = {
+  question: 'Before you submit your application, you must select one of the declaration check boxes.',
```

Comment:
not sure I understand why this is here

Reply:
This was a temporary hack to get the user-journey tests to pass.  I've revisited it now :)